apps/web/src/components/TrendingItemSuggestion/TrendingItemSuggestion.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import TrendingItemSuggestion from './TrendingItemSuggestion';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import './TrendingItemSuggestion.scss';
+import { metaData } from './TISmetaData';
+
+export default {
+  title: 'Components/TrendingItemSuggestion',
+  component: TrendingItemSuggestion,
+  argTypes: {
+    metaData: { control: 'object' },
+    handleClick: { action: 'clicked' },
+    isLoading: { control: 'boolean' },
+  },
+} as ComponentMeta<typeof TrendingItemSuggestion>;
+
+const Template: ComponentStory<typeof TrendingItemSuggestion> = (args) => (
+  <TrendingItemSuggestion {...args} />
+);
+
+export const Default = Template.bind({});
+Default.args = { metaData: metaData, isLoading: false };
+
+export const Loading = Template.bind({});
+Loading.args = { metaData: metaData, isLoading: true };
```

Comment:
Remember to use the spread operator on your `Default.arg`. I find this to be the most flexible and consistent pattern. 


```suggestion
Loading.args = { ...Default.args, isLoading: true };
```